SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialStaticComponentView.cpp

```diff
@@ -30,10 +30,10 @@ Worker_Authority USpatialStaticComponentView::GetAuthority(Worker_EntityId Entit
 	return WORKER_AUTHORITY_NOT_AUTHORITATIVE;
 }
 
-// TODO UNR-640 - Need to fix for authority loss imminent
 bool USpatialStaticComponentView::HasAuthority(Worker_EntityId EntityId, Worker_ComponentId ComponentId) const
 {
-	return GetAuthority(EntityId, ComponentId) == WORKER_AUTHORITY_AUTHORITATIVE;
+	return GetAuthority(EntityId, ComponentId) == WORKER_AUTHORITY_AUTHORITATIVE
+		|| GetAuthority(EntityId, ComponentId) == WORKER_AUTHORITY_AUTHORITY_LOSS_IMMINENT;
```

Comment:
Any specific reason for this change?

Reply:
only that the function was wrong I guess

Reply:
happy to change back as we don't plan on using soft handover, but noticed it and thought it was worth a drive by

Reply:
Yeah I'd say it's worked till now, lets leave it as is and remove it entirely with proper user-space load balancing.

Reply:
ok